qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/VisitTypeListPage.java

```diff
@@ -15,13 +15,13 @@
 import org.openmrs.contrib.qaframework.helper.Page;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.ui.ExpectedConditions;
 
 public class VisitTypeListPage extends AdminManagementPage {
 
 	private static final By ERROR = By.cssSelector("span.error");
 	private static final By VISIT_TYPE_LIST = By.cssSelector("#content form table tbody tr td");
 	private static final By RETIRED_VISIT_TYPE_LIST = By.cssSelector("#content form table tbody tr td del");
+	private static final By ALREADY_SAVED_VISIT_TYPE = By.cssSelector("#content > form > table > tbody > tr:nth-child(3) > td:nth-child(1) > a");
```

Comment:
can we use a better element reference for `ALREADY_SAVED_VISIT_TYPE `?

Reply:
Any suggestion @kdaud

Reply:
Hello @Muta-Jonathan does the element  ALREADY_SAVED_VISIT_TYPE get picked if you remove the angle brackets (>) from the selector?

Reply:
Retrieving the element through nth-child(`x`) value isn't a good ideal, the element can easily break when UI changes comes in and we need to use locators that can hardly break. You can use another cssSelector() for the element or id() or className() or linkText()

Reply:
@Muta-Jonathan  you could probably use a different element.